compiler/src/dmd/target.d

```diff
@@ -292,6 +292,8 @@ void addCppRuntimePredefinedGlobalIdent(const ref TargetCPP cpp)
     }
 }
 
+package immutable string[] obj_extensions = ["obj", "o"];
```

Comment:
This array makes it look like an easily extensible list, but the meaning of indices 0 and 1 are hardcoded in the places that actually use this array. That makes the indirection somewhat useless, it's clearer to just use the "obj" and "o" strings directly.